rasa/cdu/commands.py

```diff
@@ -130,3 +130,11 @@ class ErrorCommand(Command):
     """A command to indicate that the bot failed to handle the dialogue."""
 
     command: str = "error"
+
+
+@dataclass
+class UpdateUserProfileCommand(Command):
+    """A command to indicate that the bot should update the user profile."""
+
+    command: str = "update user profile"
```

Comment:
you'll also need to add it to `command_from_json` otherwise your command won't be deserialised